SQLQuery_3(homework).sql

```diff
@@ -0,0 +1,303 @@
+CREATE DATABASE ZooManagement;
```

Comment:
Not sure why the are so many files removed in this PR. Ideally, if we want to do multiple action that are not related to each other (like you did by removing some old c# code completely unrelated to this SQL exercices) two different PRs can be created. This makes the PRs waaaay easier to review. :)